### PR TITLE
Fake medieval weapons have FRAGILE_MELEE

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
```SUMMARY: Balance "Fake medieval weapons now have the FRAGILE_MELEE tag"```

#### Purpose of change
In my last PR (#25591), someone suggested using the FRAGILE_MELEE flag for fake weapons. While it was a good idea, it was out of the scope of what I was trying to do, which is make mansions less overpowered for players just starting out.

#### Describe the solution
This PR adds the FRAGILE_MELEE flag to all fake medieval weaponry I could find.

#### Describe alternatives you've considered
Not doing this at all. I personally disagree with how the FRAGILE_MELEE flag was applied to early game weaponry only. Right now, it is used in three items: The forked spear, the knife spear, and the glass shiv. I don't want to turn this into a rant about FRAGILE_MELEE, and if I thought it was without merit, I wouldn't be using it here.

I also don't know if this is really necessary for balance reasons. After all, these weapons are already pretty weak compared to their pretty powerful battle-ready counterparts. This is more for flavor than anything else, and if enough people feel this isn't necessary, I'll drop it.

### CODE REVIEW

**Comment by Inglonias** [comment]
Jenkins rebuild

**Comment by WizardOfOoo** [comment]
While you are at it could you up the damage of the mace and morningstar?  Two damage is worse then a rock.

File: data/json/items/melee.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
     "price": 12500,
     "material": "wood",
     "techniques": [ "RAPID", "WBLOCK_2", "SWEEP" ],
-    "flags": [ "SHEATH_SWORD" ],
+    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee.json]: This is still at least as solid as an ordinary stick, which doesnt get fragile melee, it's just too light and perhaps balanced wrong to optimum damage.

File: data/json/items/melee.json [hunk 2]
```diff
@@ -1467,7 +1467,7 @@
     "price_postapoc": 0,
     "material": "aluminum",
     "techniques": "WBLOCK_2",
-    "flags": "SHEATH_SWORD",
+    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee.json]: There are two main things that make cheap swords bad.  One is using a steel that wont hold an edge, another is the tang of the blade not extending through the hilt.  If were assuming the are extremely cheap swords, the latter is 5he case and fragile melee is called for.

I'm fine with that, I just wanted to be specific about it.

File: data/json/items/melee.json [hunk 3]
```diff
@@ -1696,7 +1696,7 @@
     "price": 1050,
     "price_postapoc": 300,
     "material": [ "aluminum", "wood" ],
-    "flags": "NONCONDUCTIVE",
+    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee.json]: I'm not particularly familiar with cheap replica axes, but as long as the haft extends through the head and it's not just welded on, I dont see fragile_melee being justified.

File: data/json/items/melee.json [hunk 4]
```diff
@@ -2082,7 +2082,7 @@
     "looks_like": "pike",
     "techniques": [ "IMPALE", "WBLOCK_2" ],
     "qualities": [ [ "COOK", 1 ] ],
-    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE" ]
+    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE", "FRAGILE_MELEE" ]
```
Comment (kevingranade) [inline data/json/items/melee.json]: Much like the axe, I'm sceptical that even cheap fake piles would be fragile enough to merit the flag.

File: data/json/items/melee.json [hunk 5]
```diff
@@ -2116,9 +2116,9 @@
     "looks_like": "mace",
     "material": [ "aluminum", "wood" ],
     "techniques": [ "SWEEP" ],
-    "flags": [ "NONCONDUCTIVE" ],
+    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee.json]: Like the axe, it seems like its be pretty easy to make one of these robust enough to not deserve the fragile flag.

File: data/json/items/melee.json [hunk 6]
```diff
@@ -2152,9 +2152,9 @@
     "material": [ "aluminum", "wood" ],
     "techniques": [ "SWEEP" ],
     "volume": 6,
-    "bashing": 2,
+    "bashing": 8,
     "cutting": 1,
-    "flags": [ "SPEAR", "NONCONDUCTIVE" ],
+    "flags": [ "SPEAR", "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```
Comment (kevingranade) [inline data/json/items/melee.json]: If we have an indication that the head would just be soldered on, fragile is warranted, I dont know if that's typical.

File: data/json/items/melee.json (hunk 1 above)
Comment (Reclusive-reptile) [reply data/json/items/melee.json]: If it was made out of balsa I could see it being fragile.

**Pull request closed without merging**